frontend-react/src/utils/TestUtils.ts

```diff
@@ -20,4 +20,35 @@ const mockEvent = (mock?: Partial<any>) => {
     };
 };
 
-export { mockEvent };
+function conditionallySupressConsole(...matchers: string[]) {
+    const origConsole = jest.requireActual("console");
+    const jestError = jest
+        .spyOn(console, "error")
+        .mockImplementation((message: any) => {
+            if (
+                !matchers.find((matcher) =>
+                    message.toString().includes(matcher)
+                )
+            ) {
+                origConsole.error(message);
+            }
+        });
+    const jestWarn = jest
+        .spyOn(console, "warn")
+        .mockImplementation((message: any) => {
+            if (
+                !matchers.find((matcher) =>
+                    message.toString().includes(matcher)
+                )
+            ) {
+                origConsole.warn(message);
+            }
+        });
+
+    return () => {
+        jestError.mockRestore();
+        jestWarn.mockRestore();
+    };
+}
+
+export { mockEvent, conditionallySupressConsole };
```

Comment:
This is a stupid, pedantic, and useless point, but I feel like inline exports (`export function blahblahblah() { ... }`) are easier to think about than scrolling up/down in the file to make sure they're exported from a specific file.  Tiny, tiny subjective thing, though!